htdocs/public/ticket/create_ticket.php

```diff
@@ -231,6 +232,21 @@
 		$object->type_code = GETPOST("type_code", 'aZ09');
 		$object->category_code = GETPOST("category_code", 'aZ09');
 		$object->severity_code = GETPOST("severity_code", 'aZ09');
+		$object->ip = (empty($_SERVER['REMOTE_ADDR']) ? 'unknown' : $_SERVER['REMOTE_ADDR']);
```

Comment:
Getting ip with $_SERVER['REMOTE_ADDR'] is not reliable (may be modified by some proxy).
You must use 
$object->id = getUserRemoteIP()